openquakeplatform/data/shape_setup.py

```diff
@@ -0,0 +1,24 @@
+from geoserver.catalog import Catalog
+cat = Catalog("http://localhost:8080/geoserver/rest/", "admin", "geoserver")
+workspace = cat.get_workspace("geonode")
+import geoserver.util
+dir_shapefile = geoserver.util.shapefile_and_friends("/home/ubuntu/geonode/data/ghec_data_sh/ghec")
```

Comment:
The installation isn't always done as 'ubuntu'. Please make the path relative. Assuming `ubuntu` as the user running script is ok for the verifier, but not for generic application code.